cmd/kube-apiserver/app/server.go

```diff
@@ -417,13 +416,10 @@ func BuildGenericConfig(s *options.ServerRunOptions, proxyTransport *http.Transp
 		// TODO: get rid of KUBE_API_VERSIONS or define sane behaviour if set
 		glog.Errorf("Failed to create clientset with KUBE_API_VERSIONS=%q. KUBE_API_VERSIONS is only for testing. Things will break.", kubeAPIVersions)
 	}
-	externalClient, err := clientset.NewForConfig(genericConfig.LoopbackClientConfig)
```

Comment:
that's interesting the "vet" tool doesn't complain about unused variable.

Reply:
Maybe because it's used in BuildAdmissionPluginInitializer?

Reply:
oh, I see.